app/jobs/regular/post_alert.rb

```diff
@@ -3,7 +3,7 @@ class PostAlert < Jobs::Base
 
     def execute(args)
       # maybe it was removed by the time we are making the post
-      if post = Post.find_by(id: args[:post_id])
+      if post = Post.find(args[:post_id])
```

Comment:
I think it is better to raise an error for visibility then let it silently bypass the job.